src/pages/signin/LoginForm/LoginFormWide.js

```diff
@@ -90,6 +93,11 @@ class LoginFormWide extends React.Component {
                         {this.props.account.error}
                     </Text>
                 )}
+                {!_.isEmpty(this.props.account.success) && (
+                    <Text style={[styles.formSuccess]}>
+                        {this.props.account.success}
+                    </Text>
+                )}
```

Comment:
cc @jasperhuangg since we were discussing a refactor to combine the wide and narrow versions. See how we needed to update this in two places for this PR?

Reply:
Yup!! Thanks for the tag; great to know how the cleanup will help :)